sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExportersEndToEndTest.java

```diff
@@ -194,42 +192,44 @@ private static void generateLog(OpenTelemetry openTelemetry) {
             .emit();
     }
 
+    @SuppressWarnings("unchecked")
     private static void validateSpan(TelemetryItem telemetryItem) {
         assertThat(telemetryItem.getName()).isEqualTo("RemoteDependency");
         assertThat(telemetryItem.getInstrumentationKey()).isEqualTo(INSTRUMENTATION_KEY);
         assertThat(telemetryItem.getTags()).containsEntry("ai.cloud.role", "unknown_service:java");
         assertThat(telemetryItem.getTags())
             .hasEntrySatisfying("ai.internal.sdkVersion", v -> assertThat(v).contains("otel"));
         assertThat(telemetryItem.getData().getBaseType()).isEqualTo("RemoteDependencyData");
-        RemoteDependencyData actualData = (RemoteDependencyData) telemetryItem.getData().getBaseData();
-        assertThat(actualData.getName()).isEqualTo("test");
-        assertThat(actualData.getProperties())
-            .containsExactly(entry("color", "red"), entry("name", "apple"));
+        Map<String, Object> actualData = telemetryItem.getData().getBaseData().getAdditionalProperties();
```

Comment:
why `additionalProperties` instead of `properties`?

Reply:
getAdditionalProperties return a map of ["properties", Map<String, String>>]
that is what the new model produced.